example/cmd/server/main.go

```diff
@@ -31,10 +31,12 @@ func (h *randomHaberdasher) MakeHat(ctx context.Context, size *example.Size) (*e
 	if size.Inches <= 0 {
 		return nil, twirp.InvalidArgumentError("Inches", "I can't make a hat that small!")
 	}
-	return &example.Hat{
+	colors := []string{"white", "black", "brown", "red", "blue"}
+	names := []string{"bowler", "baseball cap", "top hat", "derby"}
+	return &haberdasher.Hat{
```

Comment:
Please fix this to `example.Hat`, not `haberdasher.Hat`. The current code does not compile.

Reply:
Sorry about that; fixed.